cmd/satellite/main.go

```diff
@@ -365,6 +375,44 @@ func cmdNodeUsage(cmd *cobra.Command, args []string) (err error) {
 	return generateNodeUsageCSV(ctx, start, end, file)
 }
 
+func cmdNotification(cmd *cobra.Command, args []string) (err error) {
+	ctx, _ := process.Ctx(cmd)
+	logLevel := args[0]
+	message := []byte(args[1])
+
+	var parsedLogLevel pb.LogLevel
+	switch logLevel {
+	case "INFO":
```

Comment:
what if we encapsulate this switch in another type or function?
For example
```
type LogLevel string

const (
	LogLevelInfo string = "INFO"
	LogLevelWarn string = "WARN"
	LogLevelDebug string = "DEBUG"
)

func (logLevel LogLevel) Parse() pb.LogLevel {
	switch string(logLevel) {
	case LogLevelInfo:
		return pb.LogLevel_INFO
	case LogLevelWarn:
		return pb.LogLevel_WARN
	case LogLevelDebug:
		return pb.LogLevel_DEBUG
	default:
		return pb.LogLevel_ERROR
	}
}
```

Reply:
It’s also hard to make a switch statement that does one thing. By their nature, switch statements always do N things.
Unfortunately we can’t always avoid switch statements, but we can make sure that each switch
statement is buried in a low-level type and is never repeated